apps/projects/app/components/Content/IssueDetail/IssueDetail.js

```diff
@@ -239,16 +240,7 @@ const activities = (
 
 const deadlineDistance = date => formatDistance(new Date(date), new Date())
 
-const detailsCard = ({
-  issue,
-  onReviewApplication,
-  onReviewWork,
-  onUpdateBounty,
-  onViewFunding,
-  onRequestAssignment,
-  onSubmitWork,
-  onAllocateSingleBounty,
-}) => {
+const detailsCard = ({ issue }) => {
```

Comment:
Remove curly brackets here since you're already de-structuring in IssueDetail.

Reply:
Oh, weird – note that it was doing that before, though! I guess this never worked?

Reply:
Oh, I see – `IssueDetail` used to receive `issue`, now it destructures `({ issue })`.

But it always _received_ `issue`, [not destructured](https://github.com/AutarkLabs/planning-suite/pull/1010/files#diff-1e61190978ae534ae80fe1f071b7e785L427)! Confusing.

Good catch, though. I'm still not sure if/why this worked before.

Reply:
Ok, right. `IssueDetail` should have called its args `props`, but it confusingly called them `issue`. And its `propTypes` are wrong – they should be nested inside `issue`. I'll clean this up. Thanks again!